automated install/basic-install.sh

```diff
@@ -1772,9 +1772,22 @@ finalExports() {
     echo "INSTALL_WEB_INTERFACE=${INSTALL_WEB_INTERFACE}"
     echo "LIGHTTPD_ENABLED=${LIGHTTPD_ENABLED}"
     echo "CACHE_SIZE=${CACHE_SIZE}"
-    echo "DNS_FQDN_REQUIRED=true"
-    echo "DNS_BOGUS_PRIV=true"
     }>> "${setupVars}"
+
+    # if this runs the first time (variable not set yet) set the varibale to true otherwiese keep the old value
+    if [[ ! -v DNS_FQDN_REQUIRED ]]; then
```

Comment:
`-v` has been added in `bash 4.2`. Are we sure this is not another round of the gravity error we've just seen with the last release? (this was a bash v4.4 feature, though)

Reply:
Bash 4.2 is safe to use, every supported distro is 4.3 or above. But the suggested bashisms below are good to use instead. I'm not a huge fan of bashisms but I guess we might as well use them while we depend on bash as the shell.

Reply:
Is `! -v` functionally equivalent to `-n` ? Might be moot, as a I see you've made a different suggestion all together below.

Reply:
@PromoFaux 
` -v` checks for unset variable, `-n` for non-empty string.